.storybook/preview.js

```diff
@@ -3,6 +3,25 @@ import 'cypress-storybook/react';
 import * as dedent from 'dedent';
 import './docs-root.css';
 
+// This patches globals set up by cypress-storybook to work around its usage of the deprecated
+// forceReRender API that no longer works with storyStoreV7
+// https://github.com/NicholasBoll/cypress-storybook/issues/46
+// https://github.com/NicholasBoll/cypress-storybook/blob/aa9bc19d66c1cd6c6e25b93cddecb1d4a7241f2f/react.js
+// @ts-ignore
+const realSetCurrentStory = window.__setCurrentStory;
```

Comment:
**suggestion:** would be nice to wrap this into a function that express the intent


```js
makeCypressStorybookWorkWithStoryStoreV7(window);

/**
* provide docs....
*/
function makeCypressStorybookWorkWithStoryStoreV7(win){
  const realSetCurrentStory = win.__setCurrentStory;
  win.__setCurrentStory = setCurrentStoryOverride;

 function setCurrentStoryOverride(categorization, story) {
  try {
    realSetCurrentStory(categorization, story);
  } catch (e) {
    if (!(typeof e.message === 'string' && e.message.includes('was removed in storyStoreV7'))) {
      throw e;
    }
  }
};

}
```

Reply:
@ecraig12345 ^^ this was her change, i'll defer to her.